demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -346,6 +347,13 @@ def name_does_not_contain_excluded_word(self, pack_name: str) -> bool:
         lowercase_name = pack_name.lower()
         return not any(excluded_word in lowercase_name for excluded_word in EXCLUDED_DISPLAY_NAME_WORDS)
 
+    def _is_empty_dir(self, dir_path: Path) -> bool:
+        return dir_path.stat().st_size == 0
```

Comment:
cool, but may cause issues when using on a symlink.
imo using `if any dir_path.glob("*")` or similar (please test) is preferred.

Reply:
Irrelevant files.